<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ResponsablePanel</string>
  </property>
  <widget class="QPushButton" name="PayerProducteur">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>170</y>
     <width>200</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string>PayerProducteur</string>
   </property>
  </widget>
  <widget class="QListWidget" name="SousList2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>300</width>
     <height>640</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="SousList">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>300</width>
     <height>640</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Rembourser">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>440</y>
     <width>200</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string>Rembourser</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReporterReports">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>200</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string>ReporterErreur</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumProducteur">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumConsommateur">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>400</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RecruterProducteur">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>290</y>
     <width>200</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string>RecruterProducteurs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ValiderPC">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>530</y>
     <width>200</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string>ValiderCreationPC</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumPC">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>250</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
